docs/topics/salt-version-support-lifecycle.rst

```diff
@@ -26,13 +26,13 @@ Version support lifecycle
 
   * - 3007 STS
     - March 6, 2024
-    - January, 2025
-    - January, 2025
+    - January, 2026
+    - January, 2027
```

Comment:
I want to confirm this on the STS:

- Are we certain we are wanting to extend 3007 STS the same way we are extending 3006 LTS? Since 3008 LTS is planned for Q1 2025? Since I believe the plan was to have STS no longer supported once the next LTS is released?

> Salt Project is committed to providing active support for the current LTS release. There will always be at least one LTS release at a time. If unforeseen circumstances cause delays in the timing of the next LTS release, support for the current LTS release will be extended as needed.
>
> STS versions of Salt are actively supported until the date of the next LTS or STS release.